### PR TITLE
Add Kill Bounties checkbox to RA lobby options

### PR DESCRIPTION
**Pull request by pchote**
This reimplements #14245 in a more generic way.  Depends on #14359.

### CODE REVIEW

**Comment by reaperrr** [comment]
Some RA maps don't pass Travis with this.

**Comment by pchote** [comment]
Fixed (hopefully).  The issue was that I added a suffix to the inherited `GivesBounty`, which broke the maps that tried to remove it.

**Comment by pchote** [comment]
Updated and ready to review.  I chose to leave the bounty option enabled on the coop missions, Fort Lonestar, and Training Camp.

**Comment by MustaphaTR** [comment]
Needs Rebase.

**Comment by Arular101** [comment]
👍  Works as advertised. Can't say much about the code, but I did check the yaml files, which looks good to me.

I'm not sure if this is because it needs a rebase, but the Debug Menu checkbox is now the first checkbox.

**Comment by pchote** [comment]
Rebased.

**Comment by Arular101** [comment]
👍

File: OpenRA.Mods.Common/Traits/World/LobbyPrerequisiteCheckbox.cs [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+ */
+#endregion
+
+using System.Collections.Generic;
+using System.Linq;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	[Desc("Enables defined prerequisites at game start for all players if the checkbox is enabled.")]
```
Comment (reaperrr) [inline OpenRA.Mods.Common/Traits/World/LobbyPrerequisiteCheckbox.cs]: Nice, we could use this later to toggle Firestorm stuff in TS.

**Review by reaperrr — APPROVED** [review]

**Comment by reaperrr** [comment]
https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/ee35214eaa975ed50f2c3a05badfe569f8c22761...94899507327ac6b0030a270b70a735229d7058bf

**Pull request merged**